tox.ini

```
@@ -0,0 +1,38 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py{35}-django{22}, py{38}-django{22,30}, py38-quality
+
+[testenv:pyNM-djangoNM]
+setenv =
+    SELENIUM_BROWSER=chrome
+
+passenv =
+    *
+
+deps =
+    -rtest-requirements.txt
+    django22: Django==2.2.13
+    django30: Django>=3.0,<3.1
+
+commands_pre =
+    pip install -r {envdir}/src/xblock-sdk/requirements/base.txt
+    pip install -r {envdir}/src/xblock-sdk/requirements/test.txt
+
+    # upgrade both bok-choy and selenium to avoid BrokenPromise Error
+    # caused due to deprecated chromeOptions issue
+    pip install selenium==3.141.0 bok-choy==1.1.1
+
+commands =
+    python run_tests.py
+
+[testenv:py38-quality]
+deps =
+    requests
+    pylint
+
+commands =
+    pylint --rcfile=pylintrc xblock_skytap
```

Awesome, thanks for simplifying the pylint conf and fixing those issues.

Could you reinstate the pylint checks for the `tests` dir and fix/ignore those errors too?

```suggestion
    pylint --rcfile=pylintrc xblock_skytap tests
```

---

You're right @pomegranited, sorry about that, I keep forgetting about linting the tests too.

---

@pomegranited I had to do some `# pylint: disable=` comments. Sometimes mocks would cause linting errors... Anyway, it is fixed now.